#!/usr/bin/env python
# -*-coding:utf-8-*-
PACKAGE = "media_tools"

from dynamic_reconfigure.parameter_generator_catkin import * 
gen = ParameterGenerator()

switch_exp_enum = gen.enum([
    gen.const("NONE", int_t, 0, "NONE"),
    gen.const("START", int_t, 1, "START"),
    gen.const("FINISH", int_t, -1, "FINISH"),
    gen.const("STOP", int_t, 2, "STOP")
], "switch_exp_enum")

gen.add("is_debug", bool_t, 0, "Is Debug", True)

exit(gen.generate(PACKAGE, "voice_play_node", "VoicePlay"))